sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedReceiptItem.cs

```diff
@@ -5,15 +5,17 @@ namespace Azure.AI.FormRecognizer.Models
 {
     public class ExtractedReceiptItem
     {
-        public ExtractedReceiptItem(string name, int quantity, float totalPrice)
+        public ExtractedReceiptItem(string name, int quantity, float price, float totalPrice)
```

Comment:
ctor parameter types should be the same as property types (include nullablility)